fiducial_slam/src/map.cpp

```diff
@@ -177,8 +177,11 @@ Map::Map(ros::NodeHandle &nh) : tfBuffer(ros::Duration(30.0)){
 
     listener = make_unique<tf2_ros::TransformListener>(tfBuffer);
 
-    posePub = ros::Publisher(
+    robotPosePub = ros::Publisher(
           nh.advertise<geometry_msgs::PoseWithCovarianceStamped>("/fiducial_pose", 1));
+    cameraPosePub = ros::Publisher(
+          nh.advertise<geometry_msgs::PoseWithCovarianceStamped>("/camera_pose", 1));
```

Comment:
I think the topic `camera_pose` is not great. Maybe `fiducial_camera_pose` or put in int he `/fiducials` namespace.

Reply:
Good point.